__tests__/app.test.js

```diff
@@ -242,6 +242,47 @@ describe("/api/reviews/:review_id/comments", () => {
       });
   });
 });
+describe("/api/reviews/:review_id/comments", () => {
+  test("201: responds with the posted comment", () => {
+    const post = {
+      username: "mallionaire",
+      body: "Percy loves playing chess.",
+    };
+    return request(app)
+      .post("/api/reviews/2/comments")
+      .send(post)
+      .expect(201)
+      .then((response) => {
+        const comment = response.body.comment[0];
+        expect(comment.hasOwnProperty("comment_id")).toBe(true);
+        expect(comment.hasOwnProperty("body")).toBe(true);
+        expect(comment.hasOwnProperty("review_id")).toBe(true);
+        expect(comment.hasOwnProperty("author")).toBe(true);
+        expect(comment.hasOwnProperty("votes")).toBe(true);
+        expect(comment.hasOwnProperty("created_at")).toBe(true);
+        expect(comment.review_id).toBe(2);
+        expect(comment.author).toBe("mallionaire");
+        expect(comment.body).toBe("Percy loves playing chess.");
+        expect(comment.comment_id).toBe(7);
+      });
+  });
+  test("404: responds with 404 and an error message when passed a review id that does not exist", () => {
+    const post = {
+      username: "mallionaire",
+      body: "Percy loves playing chess.",
+    };
+    return request(app)
+      .post("/api/reviews/999/comments")
+      .send(post)
+      .expect(404)
+      .then((response) => {
+        expect(response.body.message).toBe(
+          "Sorry, there is no review with that id. Please try again."
+        );
+      });
+  });
+});
+
```

Comment:
we want to add some more tests, think about these situations:
-sending a comment without a body
-username of person sending comment isn't valid (they're not logged in)

if you're adventurous:
-right now now we're assuming that the comment has been successfully added to the database because we receive a 201 response.  However, what IF your model was broken, not adding to db but still sending a 201 back?  How could we check with our test that the comment has been added correctly?